.github/actions/run-integration-tests/action.yml

```diff
@@ -35,6 +40,9 @@ runs:
 
         tar zxvf /home/runner/work/inspektor-gadget/kubectl-gadget-linux-amd64.tar.gz
         mv kubectl-gadget kubectl-gadget-linux-amd64
+         
+        echo "IntegrationTestsJob: Loading the image (gadget-image-default-linux-amd64.tar) in minikube"
+        minikube image load /home/runner/work/inspektor-gadget/gadget-image-default-linux-amd64.tar
```

Comment:
What if this is not running on minikube? As in https://github.com/kinvolk/inspektor-gadget/blob/3df3dfd34a3e50afcc7dfe9bd0a3fcfe581fe349/.github/workflows/inspektor-gadget.yml#L401-L411

Reply:
I [changed](https://github.com/kinvolk/inspektor-gadget/pull/871/files#diff-bda633e49a0d242b6842493c3896d4f766f361166fa4c08e1c2e01d827544cadR229) it to push the images in those cases but probably need to add an extra check to `check-secret` to make sure  we don't hit any corner case.